packages/utilities/src/IClassNames.ts

```diff
@@ -1 +0,0 @@
-export type IClassNames<T> = { [key in keyof T]: string };
```

Comment:
I would like to major bump `@uifabric/utilities` because of this change. IClassNames simply doesn't make sense now (semantically wrong, in addition to structurally wrong) because it doesn't just have classnames...